eng/Versions.props

```diff
@@ -221,7 +221,7 @@
     <CastleCorePackageVersion>4.2.1</CastleCorePackageVersion>
     <FSharpCorePackageVersion>4.2.1</FSharpCorePackageVersion>
     <GoogleProtobufPackageVersion>3.8.0</GoogleProtobufPackageVersion>
-    <GrpcAspNetCorePackageVersion>0.1.22-pre3</GrpcAspNetCorePackageVersion>
+    <GrpcAspNetCorePackageVersion>0.2.23-dev201908160701</GrpcAspNetCorePackageVersion>
```

Comment:
The gRPC package for preview 9 is in the process of being packaged/published. This will be updated to it when ready.